docs/userManual.md

```diff
@@ -469,9 +501,9 @@ the current state of the display.
 
 * Filter...: this option will pop-up a dialog window that allows the user
   to filter the data in the selected column (this option requires only
-  one column to be selected).  The user can specify a value to be
-  sought, and a comparison, indicating whether filtering keeps values
-  identical or different to the specified one.
+  one column to be selected).  The user can specify a value/pattern to be
+  searched for (regular expressions are permitted), and also whether the
```

Comment:
I would not be so succinct. You can say that the pattern is interpreted as a regular expression using Java syntax if the next checkbox is selected.